charts/zwavejs2mqtt/2.0.0/questions.yaml

```diff
@@ -0,0 +1,522 @@
+                  type: string
+                  default: "config"
+                  required: true
+                  editable: false
+                  hidden: true
+              - variable: mountPath
+                label: "Mount path inside the container"
+                schema:
+                  type: path
+                  default: "/usr/src/app/store"
+                  editable: false
+              - variable: hostPathEnabled
+                label: "Config - Enable Host Path"
+                description: "something about this here"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "Config Host Path"
+                      description: "Host path to store config"
+                      schema:
+                        type: hostpath
+                        required: true
+        # Z-Wave device -----------------
+        - variable: zwave
+          label: "USB Z-Wave Device"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+                  hidden: true
+                  editable: false
+              - variable: setPermissions
+                label: "Automatic Permissions"
+                description: "Automatically set permissions on install"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+              - variable: name
+                label: "Mountpoint Name"
+                schema:
+                  type: string
+                  default: "ZWave"
+                  required: true
+                  editable: true
+              - variable: emptyDir
+                label: "emptyDir"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+                  editable: false
+              - variable: mountPath
+                label: "Path to mount device inside the container"
+                description: "Preferably the same as the path to the device path on the host"
+                schema:
+                  type: path
+                  required: true
+                  default: "/dev/ttyUSB0"
+                  editable: true
+              - variable: hostPathEnabled
+                label: "host Path Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: true
+              - variable: hostPath
+                label: "Path to Z-Wave device on host"
+                schema:
+                  type: hostpath
+                  required: true
+                  default: "/dev/ttyUSB0"
```

Comment:
This is fixed-up, the install completes and I can see / select the Z-Wave in the app. 

```suggestion
        # Z-Wave device -----------------
        - variable: zwave
          label: "USB Z-Wave Device"
          schema:
            type: dict
            attrs:
              - variable: hostPathEnabled
                label: "Enable Host Path"
                schema:
                  type: boolean
                  default: true
                  hidden: true
              - variable: setPermissions
                label: "Automatic Permissions"
                schema:
                  type: boolean
                  default: false
                  hidden: true
              - variable: name
                label: "Mountpoint Name"
                schema:
                  type: string
                  default: "zwave"
                  hidden: true
              - variable: emptyDir
                label: "Empty Dir"
                schema:
                  type: boolean
                  default: false
                  hidden: true
              - variable: mountPath
                label: "Path to mount Z-Wave device inside the container"
                description: "Preferably the same as the path to the device path on the host"
                schema:
                  type: path
                  default: "/dev/ttyACM0"
                  required: true
              - variable: hostPath
                label: "Path to Z-Wave device on host"
                schema:
                  type: path
                  default: "/dev/ttyACM0"
                  required: true
```

Now the issue is that a non-root user does not permission to access the Z-Wave device, so even thought it shows up, it still doesn't work. I also tried setting `setPermissions` to `default: true` but that does not help in this instance. 
```
Driver: Error: Permission denied, cannot open /dev/ttyACM0
```

For access to the Z-Wave device, the user running the app must either be `root` or a member of the `dailout` group - I tried setting PUID/PGID to `0`, but the app will not deploy. There are no visible errors, it just stays on deploying

Reply:
@tprelog can you try manually adding the apps user to the dailout group?
If that fixes it I'll make sure it gets merged into TrueNAS.

Reply:
Can you also explain what you changed in above suggestion?

Reply:
I also notice the app version is still at `1.1.1` but any option to change this has been completely removed from the questions - I know I'm still confused on how this is supposed work, but the current version is `2.3.0`

Reply:
ix_values and test_values container the version, which will be auto updated by renovate.

Reply:
I've manually updated it for you just now :)

Reply:
I've updated this suggestion and added revised details from below